docs/src/api/class-page.md

```diff
@@ -2059,7 +2059,7 @@ Shortcut for main frame's [`method: Frame.querySelector`].
 - returns: <[Array]<[ElementHandle]>>
 
 The method finds all elements matching the specified selector within the page. If no elements match the selector, the
-return value resolves to `[]`.
+return value resolves to `[]`. To wait for at least an element on the page, use [`method: Page.waitForSelector`].
```

Comment:
I don't think this helps, because there is no way to wait for all elements matching the selector. We can refer to the `querySelector` for a single-element version though. WDYT?

Reply:
sounds good, updated the PR.